resources/views/api/emails/voucher_history_email_textonly.blade.php

```diff
@@ -6,9 +6,9 @@
 
 You've requested a copy of {{ $trader }}'s voucher payment records for your reference, which is attached to this email.
 
-[xXx] The file includes payment records from {{ $date }}@isset($max_date) to {{ $max_date }}. @endisset
-[xXx] Total Vouchers: {{ count($vouchers) }}
-[xXx] Total Value: £{{ count($vouchers) }}
+The file includes payment records from {{ $date }}@isset($max_date) to {{ $max_date }}. @endisset
+Total Vouchers: {{ count($vouchers) }}
+Total Value: £{{ count($vouchers) }}
 
 The attached file is best viewed through a spreadsheet program such as Microsoft Excel, LibreOffice Calc or Google Sheets. If you have any problems with opening or downloading it, please email <a href="mailto:arc@neontribe.co.uk">arc@neontribe.co.uk</a>.
```

Comment:
same comment re. email (I know it's not part of this PR but might as well be - or card as a bug)